### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by chunyang-wen**

### CODE REVIEW

File: deepspeed/pt/deepspeed_config_utils.py [hunk 1]
```diff
@@ -8,18 +8,12 @@
...
     """Reject duplicate keys."""
-    d = {}
-    for k, v in ordered_pairs:
-        if k in d:
-            raise ValueError("Duplicate key in DeepSpeed config: %r" % (k, ))
-        else:
-            d[k] = v
+    d = dict((k, v) for k, v in ordered_pairs)
+    if len(d) != len(ordered_pairs):
+        raise ValueError("Duplicate key in DeepSpeed config: %r" % (k, ))
```
Comment (ShadenSmith) [inline deepspeed/pt/deepspeed_config_utils.py:18]: I believe the error message needs to be updated after this change. `k` will no longer contain the duplicate key.

Comment (chunyang-wen) [reply deepspeed/pt/deepspeed_config_utils.py:18]: Oops. I missed that.

**Comment by ShadenSmith** [comment]
Will merge after tests pass.

**Comment by ShadenSmith** [comment]
Thanks for your contribution to DeepSpeed!

**Pull request merged**